src/Microsoft.DotNet.ProjectModel/Utilities/FrameworksExtensions.cs

```diff
@@ -56,5 +56,12 @@ public static string GetFrameworkString(this NuGetFramework self)
             }
             return name + "-" + frameworkName.Profile;
         }
+        
+        public static bool IsPackageBased(this NuGetFramework self)
```

Comment:
IMO this is a very confusing method name, especially since there is already a property on NuGetFramework with the same name.  How would consumers know which to call? This extension method or the property?